src/pydata_sphinx_theme/assets/styles/variables/_icons.scss

```diff
@@ -10,4 +10,6 @@
   --pst-icon-exclamation-circle: "\f06a"; // fas fa-exclamation-circle
   --pst-icon-times-circle: "\f057"; // fas fa-times-circle
   --pst-icon-lightbulb: "\f0eb"; // fas fa-lightbulb
+  --pst-icon-primary-sidebar: "\f0c9"; // fas fa-bars
```

Comment:
I think that we should use a different icon for the buttons so they don't confuse people (and at the least for the right sidebar).

How about either:

1. On the right, use `fa-list`?

   ![image](https://user-images.githubusercontent.com/1839645/174749945-4a802710-85f4-4990-ba18-e9ef50d8584c.png)
2. Use `fa-outdent` on the right, and horizontally flipped `fa-outdent` on the left (https://fontawesome.com/v5/icons/outdent?s=solid). This is similar to Furo as well:

   ![image](https://user-images.githubusercontent.com/1839645/174752690-381d2c01-7126-464b-9e6f-7b8ae7cc07d1.png)

Reply:
[2a4d0cd](https://github.com/pydata/pydata-sphinx-theme/pull/726/commits/2a4d0cd4ab311cd15912054977963c34108c5e2c)